### PR TITLE
Removed pre-fills

### PR DESCRIPTION
**Pull request by matt712**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME](ADD URL HERE)

##### Before Reviewing
- [ ]  Have you pulled the branch down?
- [ ]  Have you assigned yourself to the PR?
- [ ]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [X]  Have you run the tests?
- [X]  Have you run the journey tests?
- [X]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [ ]  Have you checked the PR Builder passes?
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

File: app/controllers/dividends/UkDividendsAmountController.scala [hunk 1]
```diff
@@ -64,11 +64,11 @@ class UkDividendsAmountController @Inject()(
 
     (dividendsPriorSubmissionSession, checkYourAnswerSession) match {
       case (Some(submission@DividendsPriorSubmission(Some(prior), _)), _) =>
-          Ok(view(UkDividendsAmountForm.ukDividendsAmountForm().fill(previousAmount.getOrElse(prior)), Some(submission), taxYear, previousAmount))
+          Ok(view(UkDividendsAmountForm.ukDividendsAmountForm, Some(submission), taxYear, previousAmount))
       case (None, Some(cya)) =>
         Ok(view(cya.ukDividendsAmount.fold(
           UkDividendsAmountForm.ukDividendsAmountForm()
-        )(amount => UkDividendsAmountForm.ukDividendsAmountForm().fill(amount)), taxYear = taxYear))
+        )(amount => UkDividendsAmountForm.ukDividendsAmountForm), taxYear = taxYear))
```
Comment (michael-rob-poole) [inline app/controllers/dividends/UkDividendsAmountController.scala:71]: If there is cookie session data and no prior I think the box should be pre-filled as there is no paragraph giving the user feedback

File: app/controllers/dividends/UkDividendsAmountController.scala [hunk 2]
```diff
@@ -64,16 +64,11 @@ class UkDividendsAmountController @Inject()(
...
-        }
-        else {
-          Ok(view(UkDividendsAmountForm.ukDividendsAmountForm().fill(previousAmount.getOrElse(prior)), Some(submission), taxYear, previousAmount))
-        }
+          Ok(view(UkDividendsAmountForm.ukDividendsAmountForm, Some(submission), taxYear, previousAmount))
       case (None, Some(cya)) =>
         Ok(view(cya.ukDividendsAmount.fold(
           UkDividendsAmountForm.ukDividendsAmountForm()
-        )(amount => UkDividendsAmountForm.ukDividendsAmountForm().fill(amount)), taxYear = taxYear))
+        )(amount => UkDividendsAmountForm.ukDividendsAmountForm), taxYear = taxYear))
```
Comment (michael-rob-poole) [inline app/controllers/dividends/UkDividendsAmountController.scala:71]: I think box should be filled here. The paragraph doesn't give the user feedback here

Comment (BenCosfordHMRC) [reply app/controllers/dividends/UkDividendsAmountController.scala:71]: agreed

**Comment by BenCosfordHMRC** [comment]
No longer required.

**Pull request closed without merging**